npm/ng-packs/packages/account-config/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
+  }
+}
```

Comment:
Why we need the `tsconfig.lib.prod.json`?

Reply:
You can add `enableIvy` property to `tsconfig.lib.json`. We don't separate the our packages prod or dev.

Reply:
This file is created by `ng upgrade` itself and [angular guidelines](https://angular.io/guide/ivy#maintaining-library-compatibility) are not recommending to publish libs with Ivy. IMO, we could go with Ivy enabled all the time for abp libs but I wasn't comfortable to make this decision myself.